components/pipeline-service/development/main-pipeline-service-configuration.yaml

```diff
@@ -1960,6 +1960,136 @@ spec:
                       requests:
                         cpu: 200m
                         memory: 200Mi
+        pipelines-as-code-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-controller
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 500Mi
+                      requests:
+                        cpu: 500m
+                        memory: 500Mi
+        pipelines-as-code-watcher:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-watcher
+                    resources:
+                      limits:
+                        cpu: "1"
+                        memory: 4.5Gi
+                      requests:
+                        cpu: "1"
+                        memory: 4.5Gi
+        pipelines-as-code-webhook:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-webhook
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 400Mi
+                      requests:
+                        cpu: 500m
+                        memory: 400Mi
+        pipelines-console-plugin:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pipelines-console-plugin
+                    resources:
+                      limits:
+                        cpu: 100m
+                        memory: 35Mi
+                      requests:
+                        cpu: 100m
+                        memory: 35Mi
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: tekton-chains-controller
+                    resources:
```

Comment:
Those limits seem way too much IMO. As mention above, check the actual usage on the staging cluster and set accordingly. So for this one, I just checked and the average mem usage is around 250Mi with peaks up to 350, so maybe a value of 1GB should be enough here. For the cpu, the average stands to 40m with peaks up to 700m, so anything between 500m to 1 should be OK IMO.